pkg/server/config.go

```diff
@@ -70,6 +71,14 @@ func (cfg *Config) Validate() error {
 		cfg.Ref = "main"
 	}
 
+	if cfg.JWKSURICacheSeconds == "" {
+		// 4 hours default
+		cfg.JWKSURICacheSeconds = "14400"
+	}
+	if cfg.IssuerAllowlist == "" {
+		cfg.IssuerAllowlist = strings.Join([]string{config.GitHubIssuer, config.GoogleIssuer}, ",")
+	}
```

Comment:
You can declare these defaults with envconfig or in the CLI; it's a bit strange to do it here.

Reply:
Removed this section